packages/core/strapi/lib/commands/transfer/transfer.js

```diff
@@ -173,10 +178,19 @@ module.exports = async (opts) => {
 
     results = await engine.transfer();
   } catch (e) {
+    await strapi.telemetry.send('didDEITSProcessFail', getTransferTelemetryPayload(engine));
     exitWith(1, exitMessageText('transfer', true));
   }
 
-  const table = buildTransferTable(results.engine);
-  console.log(table.toString());
+  // Note: we need to await telemetry or else the process ends before it is sent
+  await strapi.telemetry.send('didDEITSProcessFinish', getTransferTelemetryPayload(engine));
+
+  try {
```

Comment:
Can this realistically fail? :thinking:

Reply:
Or is it to stay consistent with the rest of the DTS commands?

Reply:
If you mean the telemetry, yes, it always (or nearly always) fails if it's not awaited, so this is our only exception to the rule of not awaiting telemetry.

If you mean the try/catch, it's to stay consistent with the others, and they all have it because otherwise the table generation can throw an error (like if we pass it unexpected results data) but we still need to report that the transfer is successful.